Python/flowgraph.c

```diff
@@ -2229,6 +2229,19 @@ is_exit_without_lineno(basicblock *b) {
     return true;
 }
 
+
+static int
+max_label(cfg_builder *g)
+{
+    int lbl = -1;
+    for (basicblock *b = g->g_entryblock; b != NULL; b = b->b_next) {
+        if (b->b_label.id > lbl) {
+            lbl = b->b_label.id;
+        }
+    }
```

Comment:
This duplicates some code at the top of `translate_jump_labels_to_targets` that does the same thing. Should `translate_jump_labels_to_targets` just call `max_label` instead?